Project.toml

```diff
@@ -1,19 +1,19 @@
 name = "HSL"
 uuid = "34c5aeac-e683-54a6-a0e9-6e0fdc586c50"
-version = "0.2.1"
+version = "0.2.2"
 
 [deps]
 BinaryProvider = "b99e7846-7c00-51b0-8f62-c81ae34c0232"
 Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
-METIS_jll = "d00139f3-1899-568f-a2f0-47f597d42d70"
+METIS4_jll = "40b5814e-7855-5c9f-99f7-a735ce3fdf8b"
```

Comment:
Is it normal that this changed?

Reply:
I don't think so, it shouldn't have changed. I merged the main branch before adding changes of my own.

Reply:
But it’s already in main: https://github.com/JuliaSmoothOptimizers/HSL.jl/blob/main/Project.toml#L9

Reply:
We did it together @dpo to be able to use `HSL.jl` with other packages that require METIS 5. We recompile METIS 4 with an other Artifact name : https://github.com/JuliaPackaging/Yggdrasil/pull/2755